scripts/test/Calibration/test_ideal_tube.py

```diff
@@ -0,0 +1,38 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+
+import numpy as np
+from numpy.testing import assert_allclose
+import unittest
+from Calibration.ideal_tube import IdealTube
+
+
+class TestIdealTube(unittest.TestCase):
+    def setUp(self):
+        tube_length = 0.003278125 * 256  # assume a tube of 256 pixels, each about 3 mm tall
```

Comment:
If 3 mm in height, why do you use 0.003278125?

Reply:
I wrote _about_ 3mm. The wording is imprecise